src/js/jsx/shared/Dialog.jsx

```diff
@@ -358,8 +360,10 @@ define(function (require, exports, module) {
         render: function () {
             var children,
                 globalClass = (this.props.position === POSITION_METHODS.CENTER) ? "dialog__center" : "dialog__target",
+                // We reverse the pointerAbove/Below classes as they remove the dialog pointer rather than add it
                 classes = classnames(globalClass, this.props.className || "",
-                    { "placed-above": this.state.placedAbove }),
+                    { "pointer-above": !this.state.pointerBelow },
+                    { "pointer-below": !this.state.pointerAbove }),
```

Comment:
Hmm, this is kind of confusing. Is it possible for these two sets of names to be more in sync?